src/Core/Validators/CommandResultValidator.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Bit.Core.Models.Commands;
+
+namespace Bit.Core.Validators;
+
+public static class CommandResultValidator
+{
+    public static CommandResult ExecuteValidators(Func<CommandResult>[] validators)
```

Comment:
"Run synchronous validators, then run async validators" is probably going to be a pretty common pattern. I suggest a combined interface, like `ExecuteValidatorsAsync(Func<CommandResult>[], Func<Task<CommandResult>>[])` which can just handle it all for you. That itself would be asynchronous but doesn't matter as long as it's guaranteed to run synchronous validators first.

Reply:
I do like having a method that can handle both. Plus, this will ensure that we always handle sync validators first before async validators.

There is a very small trade-off in that if you only need sync validators, you will have to call an async method, which creates a little overhead. However, the value of always ensuring sync validators run first outweighs this minor edge case.